cpu/stm32/periph/eth.c

```diff
@@ -437,6 +511,11 @@ void stm32_eth_isr_eth_wkup(void)
 
 static void stm32_eth_isr(netdev_t *netdev) {
     netdev->event_callback(netdev, NETDEV_EVENT_RX_COMPLETE);
```

Comment:
Now this is wrong.

Reply:
I have added the function `stm32_eth_get_rx_status_owned` again to be able to distinguish between the link up and a message reception.